addon/decorators/validation/is-int.js

```diff
@@ -0,0 +1,32 @@
+import { registerDecorator } from "class-validator";
+import { getOwner } from '@ember/application';
+
+export function IsInt(validationOptions) {
+  return function (object, propertyName) {
+    registerDecorator({
+      name: 'IsInt',
+      target: object.constructor,
+      propertyName: propertyName,
+      options: validationOptions,
+      validator: {
+        validate(value) {
+          const numValue = Number(value);
+          if (Number.isInteger) {
+            return Number.isInteger(numValue);
+          }
+          // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger#Polyfill
```

Comment:
It looks like this exists in all of our supported browsers, is there a specific one that I missing that would need a polyfill for this?

Reply:
internet exploder doesn't support this method. but even tho we don't support IE anymore, it costs us virtually nothing to polyfill this method. not feeling super strong about this, so if you want this gone then i'll yank it.

Reply:
It's fine with me, you're right no big deal.